dpdata/vasp/outcar.py

```diff
@@ -13,7 +13,9 @@ def system_info (lines, type_idx_zero = False) :
                 atom_names.append(_ii.split('_')[0])
             else:
                 atom_names.append(_ii)
-        elif 'NELM' in ii and nelm == None:
+        #find the line contains NELM but not initiated by other vasp_keys like NELMIN and NELMDL ...
+        #...format adjustmnet in vasp 6 request this check ; compatible with the older version
+        elif 'NELM' in ii and nelm == None and (ii.split()[0] != "NELMIN") and (ii.split()[0] != "NELMDL"):
```

Comment:
I would suggest to use `"NELM   =" in ii`, to prevent other lines with `NELM` appears before `NELM   =` in future versions

Reply:
the number of spaces between "NELM" and "=" might not be safe (currently maybe 2 or 3 not sure) 

revise into a stricter check 
expecting the following format : 
```
"anywhere_in_the_line"+"NELM"+"any_number_of_spaces"+"="+"value(data-type-check_omitted_don't-believe-it's-needed-here)"+"whatever_else"
```

Reply:
You may need to use regular expression.

Reply:
> You may need to use regular expression.

support

Reply:
> You may need to use regular expression.

Could you please suggest the change with regex ?